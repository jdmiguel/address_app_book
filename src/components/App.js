import React from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';

import HomePage from './pages/HomePage';
import SettingsPage from './pages/SettingsPage';

/**
 * Main component in charge of handling the different routes of the application
 * @component App
 */

const App = () => (
  <Router>
    <Switch>
      <Route exact path="/" component={HomePage} />
      <Route exact path="/settings" component={SettingsPage} />
    </Switch>
  </Router>
);

export default App;
