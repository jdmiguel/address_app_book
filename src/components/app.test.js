import React from 'react';
import { render } from '@testing-library/react';

import App from './App';

describe('Component: App', () => {
  it('should render', () => {
    const { container } = render(<App />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
