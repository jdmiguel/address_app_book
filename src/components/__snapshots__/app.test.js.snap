// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App should render 1`] = `
<div
  class="app"
>
  <header
    class=""
  >
    <div
      class="container"
    >
      <h1>
        ADDRESS BOOK
      </h1>
    </div>
  </header>
</div>
`;
